src/main/java/com/cloudbees/jenkins/support/impl/SystemConfiguration.java

```diff
@@ -46,8 +47,29 @@
  * System configuration data (CPU information, swap configuration, mount points,
  * kernel messages and entropy)
  */
-@Extension
-public class SystemConfiguration extends ProcFilesRetriever {
+public abstract class SystemConfiguration extends ProcFilesRetriever {
+    @Extension
+    public static class Master extends SystemConfiguration {
+        @Override
+        @NonNull
+        public String getDisplayName() {
+            return "Master system configuration (Linux only)";
```

Comment:
Sine this is only available for linux I think we should add the ability to not show it if it's on windows. I.E.

`@Override public boolean isEnabled() { return (Platform.current() == Platform.UNIX); }`